cluster/mesos/docker/docker-compose.yml

```diff
@@ -0,0 +1,139 @@
+ambassador:
+  image: cpuguy83/docker-grand-ambassador:0.9.1
+  volumes:
+  - /var/run/docker.sock:/var/run/docker.sock
+  command: "-name docker_apiserver_1"
+etcd:
+  hostname: etcd
+  image: quay.io/coreos/etcd:v2.0.10
+  ports: [ "4001:4001" ]
+  command: >
+    --listen-client-urls 'http://etcd:4001'
+    --advertise-client-urls 'http://etcd:4001'
+    --initial-cluster-state new
+mesosmaster1:
+  hostname: mesosmaster1
+  image: mesosphere/mesos:0.22.0-1.0.ubuntu1404
+  entrypoint: mesos-master
```

Comment:
Actually, this seems to be a bug in docker-compose and/or Docker 1.7. I think also the (not recommended) shell entrypoint should work in fact.